### ISSUE TITLE
TAG translation in French

### ISSUE THREAD

**Issue by kikoucalou** [scheckmedia/photils-cli#16]
Hello, 

I have made good progress on a translation of TAG into French.

I think it will be well refined and sufficiently reviewed in a few days, so that I will be able to submit it to you for sharing with the community.

Could you please let me know how to submit it to you?

**Comment by manu-p** [comment]
Hi @kikoucalou 

I got to it before I realized you'd done something. :-/

You'd have to make a pull request (PR), a priori.

The thing that bothers me is that although I've put my version as of ~/.local/share/photils/override_labels.json (made it executable, you never know...) on my Ubuntu 22.04, as mentioned [here](https://github.com/scheckmedia/photils-cli?tab=readme-ov-file#tag-translation), **photils continues to output only English keywords**...

Moreover, I can see the latest version has not moved since july 2022. This and the fact you did not get any answer make me think this might be once again a dead project.

**Comment by scheckmedia** [comment]
Yes, I agree, besides a full-time job as an ML engineer, I don't want to spend my free time in front of the computer, so side projects have to be put on hold until I find some time and, more importantly, the desire.

**Comment by manu-p** [comment]
Hi @scheckmedia thank you for your reply, which I understand perfectly!

I'm going to move my question to another topic anyway, if you can find a moment to take a look.

**Comment by manu-p** [comment]
Hi again, just submitted a PR for the french translations... Hope I did it properly, I'm not used to do things with github.
